pages/profile.js

```diff
@@ -13,10 +13,19 @@ import ProfileTasks from './../components/ProfileTasks'
 import Modal from 'react-modal'
 import React from 'react'
 import ExitBetaModal from '../components/ExitBetaModal'
+import { useSession } from 'next-auth/react'
 
 export default function Profile(props) {
   /* istanbul ignore next */
   const t = props.locale === 'en' ? en : fr
+  const { status } = useSession({
+    required: true,
+    onUnauthenticated() {
+      // The user is not authenticated, handle it here.
+      console.log('User is not logged in')
+    },
+  })
+  console.log('User Status: ', status)
```

Comment:
Can this use the existing logic within my-dashboard to check if users are logged in?
Check ./lib/auth AuthIsValid() to check if users are authenticated